package.json

```diff
@@ -26,6 +26,7 @@
     "contentlayer": "^0.3.4",
     "cookies-next": "^4.2.1",
     "copy-to-clipboard": "^3.3.1",
+    "date-fns": "^4.1.0",
```

Comment:
Is `date-fns` necessary if we also have `dayjs`?

Reply:
jr beat me to asking what it was for haha